### PR TITLE
Fix the stopwatch from time_tracking feature.

### PR DESCRIPTION
**Pull request by EthraZa**
Before, it was lagging behind the real time and was not possible to manually edit the time field.

Fix [#22919](http://www.mantisbt.org/bugs/view.php?id=22919)

### CODE REVIEW

File: js/common.js [hunk 1]
```diff
@@ -168,60 +168,68 @@ $(document).ready( function() {
...
-			}
-			if (minutes < 10) {
-				minutes = '0' + minutes;
-			}
-			if (hours < 10) {
-				hours = '0' + hours;
-			}
-			$('input[type=text].stopwatch_time').val(hours + ':' + minutes + ':' + seconds);
-			this.start();
+			$('input[type=text].stopwatch_time').val(this.zeroTime.clone().add(moment().diff(this.startTime)).format('HH:mm:ss'));
```
Comment (dregad) [inline js/common.js]: Please break the long line

**Comment by EthraZa** [comment]
Requested changes have been made.
About the "self = this", it is neede inside the function, to access the outer scope. In JS functions have it's own scopes. But I have changed to use the self var only when it's really needed.

**Comment by dregad** [comment]
Cheers. I'll test the PR again later tonight.

> About the "self = this", it is neede inside the function, to access the outer scope

Thanks for the explanation.

**Comment by dregad** [comment]
@EthraZa Thanks for your contribution

**Pull request merged**